arroyo-sql/src/lib.rs

```diff
@@ -688,3 +705,22 @@ pub fn get_test_expression(
         expected_result,
     )
 }
+
+pub fn has_duplicate_udf_names(udf_defs: &Vec<String>) -> Result<bool> {
+    let mut udf_names = HashMap::new();
+    for udf_def in udf_defs {
+        let file = syn::parse_file(udf_def)?;
```

Comment:
This will cause everything to fail if you have a single bad UDF; is that what we want?

Reply:
I've changed it to log a warning and continue